addons/website_sale_delivery/views/website_sale_delivery_templates.xml

```diff
@@ -5,7 +5,7 @@
         <xpath expr="//tr[@id='order_total_untaxed']" position="before">
             <tr id="order_delivery" t-if="website_sale_order and website_sale_order.carrier_id">
               <td class="text-right noborder text-muted"  title="Delivery will be updated after choosing a new delivery method">Delivery:</td>
-              <td class="text-xl-right noborder text-muted" >
+              <td class="text-right noborder text-muted" >
```

Comment:
Take the occasion to remove the extra space here too :)

Reply:
@qsm changes are done.
and I also have removed extra space from other places after the class :)